tickets/src/__tests__/actions/key.test.js

```diff
@@ -1,11 +1,4 @@
-import {
-  purchaseKey,
-  addKey,
-  updateKey,
-  PURCHASE_KEY,
-  ADD_KEY,
-  UPDATE_KEY,
-} from '../../actions/key'
+import { purchaseKey, setKey, PURCHASE_KEY, SET_KEY } from '../../actions/key'
```

Comment:
I replaced `addKey` and `updateKey` with a single `setKey` which combines their logic.